src/rest-api/endpoints/management-endpoint/configuration-endpoint.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Configuration Endpoint
+review:
+  comment: ''
+  date: '2023-07-17'
+  status: ok
+labels:
+  - http
+  - rest-api
+  - configuration
+toc: true
+tree_item_index: 600
+---
+
+## Get a Nuxeo instance's configuration
+
+```
+GET /management/configuration
+```
+
+### Response
+
+If successful, returns a JSON representation of all configured properties and all runtime properties with sensitive informations redacted.
```

Comment:
Expect to have questions on what is configured vs runtime props, and why runtime props duplicates configured props.

Reply:
Do you mean I should explain it here ?

Reply:
I have merged the explenation from the ticket in the sentence.